GitFyle.Core.Api.Tests.Unit/Controllers/ContributionTypes/ContributionTypesControllerTests.Exceptions.Delete.cs

```diff
@@ -167,31 +166,31 @@ public async Task ShouldReturnLockedOnDeleteIfRecordIsLockedAsync()
         }
 
         [Fact]
-        public async Task ShouldReturnFailedDependencyOnDeleteIfReferenceErrorOccursAsync()
+        public async Task ShouldReturnFailedDependencyOnDeleteIfReferenceExceptionOccursAsync()
         {
             // given
-            Guid someId = Guid.NewGuid();
+            Guid someContributionTypeId = Guid.NewGuid();
             ContributionType someContributionType = CreateRandomContributionType();
             var someInnerException = new Exception();
             string someMessage = GetRandomString();
 
-            var invalidReferenceContributionTypeException =
-                new InvalidReferenceContributionTypeException(
-                    message: someMessage,
-                    innerException: someInnerException,
-                    data: someInnerException.Data);
+            var invalidReferenceContributionTypeException = 
+                    new InvalidReferenceContributionTypeException(
+                        message: someMessage,
+                        innerException: someInnerException,
+                        data: someInnerException.Data);
 
-            var contributionTypeDependencyValidationException =
-                new ContributionTypeDependencyValidationException(
-                    message: someMessage,
-                    innerException: invalidReferenceContributionTypeException,
-                    data: invalidReferenceContributionTypeException.Data);
+            var contributionTypeDependencyValidationException = 
+                    new ContributionTypeDependencyValidationException(
+                        message: someMessage,
+                        innerException: invalidReferenceContributionTypeException,
+                        data: invalidReferenceContributionTypeException.Data);
 
-            FailedDependencyObjectResult expectedConflictObjectResult =
-               FailedDependency(invalidReferenceContributionTypeException);
+            FailedDependencyObjectResult expectedFailedDependencyObjectResult = 
+                    FailedDependency(invalidReferenceContributionTypeException);
 
-            var expectedActionResult =
-                new ActionResult<ContributionType>(expectedConflictObjectResult);
+            var expectedActionResult = 
+                    new ActionResult<ContributionType>(expectedFailedDependencyObjectResult );
```

Comment:
```suggestion
                    new ActionResult<ContributionType>(expectedFailedDependencyObjectResult);
```
remove extra space after expectedFailedDependencyObjectResult